File: problems/easy/outofStock/__tests__/outofStock.js

const inventoryFiller = require('../outofStock');
const fillInventory = require('../outofStock');

function test() {
    const result = inventoryFiller(["milk", "eggs", "bread"], ["bread", "butter", "milk", "cheese"]);
    if (JSON.stringify(result.sort()) === JSON.stringify(["butter", "cheese"].sort())) {
        console.log("✅ JS test passed");
    } else {
        console.log("❌ JS test failed");
    }
}

test();

File: problems/easy/outofStock/__tests__/outofStock.py
import sys
import os

sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from outofStock import inventoryFiller

def test():
    result = inventoryFiller(["milk", "eggs", "bread"], ["bread", "butter", "milk", "cheese"])
    if sorted(result) == sorted(["butter", "cheese"]):
        print("✅ Python test passed")
    else:
        print("❌ Python test failed")

test()
